### PR TITLE
Update Javadoc of premium SDK for azure-resourcemanager-cosmos

### PR DESCRIPTION
**Pull request by v-hongli1**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java [hunk 1]
```diff
@@ -24,13 +24,25 @@
 public interface CosmosDBAccount extends GroupableResource<CosmosManager, DatabaseAccountGetResultsInner>,
     Refreshable<CosmosDBAccount>, Updatable<CosmosDBAccount.Update>, SupportsUpdatingPrivateEndpointConnection {
 
-    /** @return indicates the type of database account */
+    /**
+     * Gets indicates the type of database account.
```
Comment (XiaofeiCao) [inline sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java:28]:
```suggestion
     * Gets the type of database account.
```

File: sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java [hunk 2]
```diff
@@ -41,103 +53,206 @@ public interface CosmosDBAccount extends GroupableResource<CosmosManager, Databa
...
+    /**
+     * Check whether write is enabled for multiple locations.
+     *
+     * @return whether write is enabled for multiple locations or not
+     */
     boolean multipleWriteLocationsEnabled();
 
-    /** @return whether cassandra connector is enabled or not. */
+    /**
+     * CHeck whether cassandra connector is enabled.
```
Comment (XiaofeiCao) [inline sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java:163]:
```suggestion
     * Check whether cassandra connector is enabled.
```

File: sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java [hunk 3]
```diff
@@ -41,103 +53,206 @@ public interface CosmosDBAccount extends GroupableResource<CosmosManager, Databa
...
+    /**
+     * Gets the current cassandra connector offer.
+     *
+     * @return the current cassandra connector offer.
+     */
     ConnectorOffer cassandraConnectorOffer();
 
-    /** @return whether metadata write access is disabled or not. */
+    /**
+     * Check whether metadata write access is disabled.
```
Comment (XiaofeiCao) [inline sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java:177]:
```suggestion
     * Checks whether metadata write access is disabled.
```

Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java]: Fixed by the [commit#a8c40e1](https://github.com/Azure/azure-sdk-for-java/pull/43524/commits/a8c40e15e3cfe24f29ba518652c5b7baa5c057b7)

**Review by XiaofeiCao — APPROVED** [review]

File: sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java [hunk 4]
```diff
@@ -41,103 +53,206 @@ public interface CosmosDBAccount extends GroupableResource<CosmosManager, Databa
...
+     *
      * @param groupName group name of private link resource
      * @return the specific private link resource group
      */
     Mono<PrivateLinkResource> getPrivateLinkResourceAsync(String groupName);
 
     /**
+     * Gets the specific private link resource group.
+     *
+     *
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java:208]: nit, one new line

File: sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java [hunk 5]
```diff
@@ -41,103 +53,206 @@ public interface CosmosDBAccount extends GroupableResource<CosmosManager, Databa
...
      * @param groupName group name of private link resource
      * @return the specific private link resource group
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java:210]:
```suggestion
     * Gets the specific private link resource.
     *
     * @param groupName group name of private link resource
     * @return the specific private link resource
```

File: sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java [hunk 6]
```diff
@@ -41,103 +53,206 @@ public interface CosmosDBAccount extends GroupableResource<CosmosManager, Databa
...
+     * @return all private link resources in the account.
+     */
     List<PrivateLinkResource> listPrivateLinkResources();
 
     /**
+     * Gets the specific private link resource group.
+     *
      * @param groupName group name of private link resource
      * @return the specific private link resource group
      */
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java:202]:
```suggestion
    /**
     * Gets the specific private link resource.
     *
     * @param groupName group name of private link resource
     * @return the specific private link resource
     */
```

File: sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java [hunk 7]
```diff
@@ -170,10 +285,16 @@ public interface CosmosDBAccount extends GroupableResource<CosmosManager, Databa
...
-    /** @param keyKind the key kind */
+    /**
+     * Regenerates secret keys.
+     *
+     * @param keyKind the key kind
+     */
     void regenerateKey(KeyKind keyKind);
 
     /**
+     * Gets a representation of the deferred computation of this call.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java:296]:
```suggestion
     * A deferred emitter that regenerates secret keys.
```

**Pull request merged**